src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -86,6 +86,10 @@
     <Compile Remove="coreclr\Math\Functions\Single\ScaleB.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" !('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' == '3.0')">
```

Comment:
What's the intent here? to be included in not 3.0?

Reply:
Should it be this:
```xml
<ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
```

Reply:
Do not depend in msbuild private properties (start with `_`)

Reply:
Sure, but I followed the existing precedent in this csproj file:
https://github.com/dotnet/performance/blob/c5e5f7e8a2eaccecb7752d1eab51c143adcd742f/src/benchmarks/micro/MicroBenchmarks.csproj#L78

Reply:
I did not catch that one.